_config.yml

```diff
@@ -14,12 +14,13 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://activate.mozilla.community" # the base hostname & protocol for your site
 twitter_username: mozparticipate
 github_username:  mozilla
+api_base_url: https://activateapi.michaelkohler.info
```

Comment:
for now running on my server, filed https://github.com/mozilla/activate.mozilla.community/issues/274 for the move to Community IT.